### PR TITLE
Documentation that Steve asked (for users)

### PR DESCRIPTION
**Pull request by tmozgach**

### CODE REVIEW

**Comment by tmozgach** [comment]
@sjackman could you please review the documentation that Steve asked, do I need add something more?

**Comment by sjackman** [comment]
My paper deadline was pushed back until Monday. I'll review these docs next weeks after my paper's submitted.

File: doc2.rmd [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+It’s under the MIT license using the Docker , an open-source platform that 
+packages up an application with all of the minimal dependencies it needs.
+
+
+## How to install Docker:
+
+[On Mac](https://docs.docker.com/docker-for-mac/install/)
+
+[On Linux](https://docs.docker.com/engine/installation/#cloud)
```
Comment (sjackman) [inline doc2.rmd]: Remove `#cloud`

File: doc2.rmd [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+[On Windows](https://docs.docker.com/docker-for-windows/install/#download-docker-for-windows)
```
Comment (sjackman) [inline doc2.rmd]: Remove `#download-docker-for-windows`

File: doc2.rmd [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# ORCA
+
+ORCA is a platform for bioinformatics analysis. It is suited for those wishing to conduct self-serve analysis using their own existing data or provides a comprehensive bioinformatics environment for education or research, making results reproducible.
+
+ORCA provides access to a menu of validated bioinformatics software and the power to use these inside a secure and private containerized environment. These private containers are backed by larger mounted storage volumes and dedicated compute nodes to ensure sufficient resources for the demanding needs of bioinformatics analysis.
+Hundreds of bioinformatics tools from Homebrew-Science are installed in the ORCA Docker image using [Linuxbrew](http://linuxbrew.sh/), the package manager for Linux. ORCA is based on version 16.04 of Ubuntu Image.
+
+It’s under the MIT license using the Docker , an open-source platform that 
+packages up an application with all of the minimal dependencies it needs.
```
Comment (sjackman) [inline doc2.rmd]: ORCA itself is available under the MIT license. The software tools that it includes are licensed under various open source licenses.

ORCA is available as a Docker image that includes hundreds of preinstalled software packages.

File: doc2.rmd [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+## How to install Docker:
+
+[On Mac](https://docs.docker.com/docker-for-mac/install/)
+
+[On Linux](https://docs.docker.com/engine/installation/#cloud)
+
+[On Windows](https://docs.docker.com/docker-for-windows/install/#download-docker-for-windows)
+
+Use the following commands in the terminal to check whether you have the Docker:
```
Comment (sjackman) [inline doc2.rmd]: Change "the Docker" to "Docker installed"

File: doc2.rmd [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+```
+docker --version
+docker --help
+```
```
Comment (sjackman) [inline doc2.rmd]: Add a blank line here.

File: doc2.rmd [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+## How to install ORCA
+Use the following commands in the terminal to download and run ORCA image:
+
+```
+docker pull bcgsc/orca:2017.08.23
+docker exec -it bcgsc/orca:2017.08.23 /bin/bash
```
Comment (sjackman) [inline doc2.rmd]: Remove `:2017.08.23` from both of these lines to ensure that they get the latest image.
Change `exec` to `run`
Remove `/bin/bash`

File: doc2.rmd [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+```
+
+## Installing software in ORCA
+[Linuxbrew](http://linuxbrew.sh/) was integrated with ORCA for easy distribution of software.
+For the installation of software through [Linuxbrew](http://linuxbrew.sh/), use the command below in the terminal:
+
+```
+brew install homebrew/science/<software>
```
Comment (sjackman) [inline doc2.rmd]: Remove `homebrew/science/`. It'll be renamed before the end of this year.

File: doc2.rmd [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+## How to install ORCA
+Use the following commands in the terminal to download and run ORCA image:
+
+```
+docker pull bcgsc/orca:2017.08.23
+docker exec -it bcgsc/orca:2017.08.23 /bin/bash
+```
+
+## Installing software in ORCA
+[Linuxbrew](http://linuxbrew.sh/) was integrated with ORCA for easy distribution of software.
```
Comment (sjackman) [inline doc2.rmd]: Change `was` to `is`
Change `distribution` to `installation`

File: doc2.rmd (hunk 7 above)
Comment (sjackman) [reply doc2.rmd]: Change `<software>` to a real world example, like `zsh`.

**Comment by sjackman** [comment]
Include link to this list of available software: http://formulae.brew.sh
Describe a few useful paths, such as
`/home/linuxbrew/.linuxbrew/bin`
and
`/home/linuxbrew/.linuxbrew/opt`
and
`/home/linuxbrew/.linuxbrew/Cellar`

**Comment by sjackman** [comment]
I think we should describe how to mount their local home directory inside the ORCA container.
```
docker run -it -V $HOME bcgsc/orca
```

File: doc2.rmd (hunk 6 above)
Comment (tmozgach) [reply doc2.rmd]: On Docker Hub the tag is `2017.08.23`, if you pull just bcgsc/orca, it will download the previous image with tag `latest`

Comment (sjackman) [reply doc2.rmd]: The latest most current image should have the tag `latest`.

**Comment by tmozgach** [comment]
```
Include link to this list of available software: http://formulae.brew.sh
```
but it is a list for homebrew for Mac? Should we add that?

**Comment by sjackman** [comment]
Yep, it's fine. Those formulae are available for both macOS and Linux.

**Comment by tmozgach** [comment]
@sjackman I didn't find exact definitions for those folders, could you please briefly describe?
```
/home/linuxbrew/.linuxbrew/bin
/home/linuxbrew/.linuxbrew/opt
/home/linuxbrew/.linuxbrew/Cellar
```

**Comment by sjackman** [comment]
`/home/linuxbrew/.linuxbrew/bin` includes the executables and scripts in your `PATH`.
`/home/linuxbrew/.linuxbrew/opt` includes symlinks to the most recent version of each software package.
`/home/linuxbrew/.linuxbrew/Cellar` includes all versions of the installed software packages.

**Comment by tmozgach** [comment]
@sjackman  thanks! I edited the text.

**Pull request merged**